src/CalcManager/Header Files/CalcEngine.h

```diff
@@ -147,6 +152,8 @@ class CCalcEngine
     NUM_WIDTH m_numwidth;                    // one of qword, dword, word or byte mode.
     int32_t m_dwWordBitWidth;                // # of bits in currently selected word size
 
+    uint64_t m_carryBit;
```

Comment:
It was easier to leave it at uint64_t since we use OR operations to set the msb in calculations and the msb and lsb are currently also handled as uint64_t